test/interface_c.cpp

```diff
@@ -0,0 +1,112 @@
+#include <gtest/gtest.h>
+
+extern "C" int trixi_version_major();
+extern "C" int trixi_version_minor();
+extern "C" int trixi_version_patch();
+extern "C" const char* trixi_version();
+extern "C" const char* trixi_version_julia();
+extern "C" const char* trixi_version_julia_extended();
+extern "C" void trixi_initialize(const char * project_directory, const char * depot_path);
+extern "C" int trixi_initialize_simulation(const char * libelixir);
+extern "C" void trixi_finalize_simulation(int handle);
+extern "C" void trixi_finalize();
+extern "C" int trixi_is_finished(int handle);
+extern "C" void trixi_step(int handle);
+extern "C" double trixi_calculate_dt(int handle);
+extern "C" int trixi_ndims(int handle);
+extern "C" int trixi_nelements(int handle);
+extern "C" int trixi_nvariables(int handle);
+extern "C" void trixi_load_cell_averages(double * data, int handle);
```

Comment:
Why not
```c++
extern "C" {
  int trixi_version_major();
  ...
}
```
?

Reply:
Because I started with
```
extern "C" int trixi_version_major();
```
and the resorted to a known solution. :blush: